index.html

```diff
@@ -269,11 +269,42 @@ <h4>Computation steps</h4>
         <li id="step1">Set the <code>root node</code> to the given element, the <code>current node</code> to the <code>root node</code>, and the <code>total accumulated text</code> to the empty string (""). If the <code>root node</code>'s role prohibits naming, return the empty string ("").</li>
         <li id="step2">Compute the text alternative for the <code>current node</code>:
           <ol>
-            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is not</strong> directly referenced by <code>aria-labelledby</code> or <code>aria-describedby</code>, nor directly referenced by a native host language text alternative [=element=] (e.g. <code>label</code> in HTML) or [=attribute=], return the empty string.
+            <li id="step2A">If the <code>current node</code> is <a class="termref">hidden</a> <strong>and is</strong>:
+            <ul>
+              <li><strong>Not</strong> part of an <code>aria-labelledby</code> or <code>aria-describedby</code> traversal, where the node directy referenced by that relation was hidden.</li>
```

Comment:
Last two words ("_was_ hidden") seem to imply either transiency (e.g. element is no longer hidden) or that the element was the direct target of the hiding (e.g. this specific element is the actually hidden node rather than being hidden as a side effect of a hidden ancestor).

Perhaps "_is_ hidden" would be better here, since I understand the intent to be "currently hidden" and we don't care why/how it was hidden? If I'm misunderstanding, more clarification of the intent would be helpful.

I can't add a diff suggestion. Perhaps because it's already merged?

Reply:
> Perhaps "is hidden" would be better here, since I understand the intent to be "currently hidden" and we don't care why/how it was hidden? 

Exactly this, I was not trying to express anything different. I should have used the present tense as in the original.